src/features/Students/StudentsMetrics/index.jsx

```diff
@@ -39,15 +45,15 @@ const StudentsMetrics = () => {
             title="New students registered"
           />
           <Card.Section>
-            <div className="card-number">{stateMetrics.newStudentsRegistered}</div>
+            <div className="card-number">{stateStudentsMetrics.numberOfEnrollments}</div>
           </Card.Section>
         </Card>
         <Card className="card-green">
           <Card.Header
             title="Classes scheduled"
           />
           <Card.Section>
-            <div className="card-number">{stateMetrics.classesScheduled}</div>
+            <div className="card-number">{stateClassesMetrics.numberOfClassesCreated}</div>
```

Comment:
If an error happens for whatever the reason, nothing is shown, which is not exactly bad, however we could handle it better, like use a fallback value like a `0` or `--` I don't know.

![Screenshot from 2024-03-05 12-33-36](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/66f4a420-6fbd-4b03-a7e8-e4cf5a218b39)

Reply:
@01001110J yes, you're right.